README.rst

```diff
@@ -82,70 +84,78 @@ PyPI wheels (pip)
 
 Lightning plugins can be installed using ``pip`` as follows
 
-.. code-block:: console
+.. code-block:: bash
 
-    $ pip install pennylane-lightning
+    pip install pennylane-lightning
 
 The above command will install the Lightning-Qubit plugin (the default since it is most broadly supported).
 In order to install the Lightning-GPU and Lightning-Kokkos (OpenMP) backends, you can respectively use the following commands:
 
-.. code-block:: console
+.. code-block:: bash
 
-    $ pip install pennylane-lightning[gpu]
-    $ pip install pennylane-lightning[kokkos]
+    pip install pennylane-lightning[gpu]
+    pip install pennylane-lightning[kokkos]
 
 
 Install from source
 ===================
 
 To build Lightning plugins from source you can run
 
-.. code-block:: console
+.. code-block:: bash
 
-    $ PL_BACKEND=${PL_BACKEND} pip install pybind11 pennylane-lightning --no-binary :all:
+    PL_BACKEND=${PL_BACKEND} pip install pybind11 pennylane-lightning --no-binary :all:
 
-where ``${PL_BACKEND}`` can be ``lightning_qubit`` (default), ``lightning_gpu`` or ``lightning_kokkos``.
+where ``${PL_BACKEND}`` can be ``lightning_qubit`` (default), ``lightning_gpu``,  ``lightning_kokkos``, or ``lightning_tensor``.
 The `pybind11 <https://pybind11.readthedocs.io/en/stable/>`_ library is required to bind the C++ functionality to Python.
 
 A C++ compiler such as ``g++``, ``clang++``, or ``MSVC`` is required.
 On Debian-based systems, this can be installed via ``apt``:
 
-.. code-block:: console
+.. code-block:: bash
```

Comment:
I think "console" is more general than "bash". I also didn't know "bash" is a valid code-block option :thinking: it's worth double-checking :)

Reply:
`console` is more general, but an issue I found is that we lose the highlighted syntax for variables or commands unless if we add `$` at the begining of the line. 

I made a couple of tests to see if this change breaks something but everything looks fine, in GitHub and Website doc. 

*using `bash`*
![image](https://github.com/user-attachments/assets/05b203e4-2c6a-40d6-a089-ed08083dd020)

*using `console`*
![image](https://github.com/user-attachments/assets/337f7dba-70e2-4c75-ac05-71c7850c3d7d)

GitHub
*using `bash`*
![image](https://github.com/user-attachments/assets/4bada1cf-75bc-4f96-a7eb-ee5085208301)
*using `console`*
![image](https://github.com/user-attachments/assets/1aa8029f-dea2-4b29-af0e-0a56448884dc)



Also, `bash` has support for *Sphinx* [ref](https://pygments.org/docs/lexers/#lexers-for-various-shells)